ReText/editor.py

```diff
@@ -106,12 +106,20 @@ def __init__(self, parent):
 		self.statistics = (0, 0, 0)
 		self.statsArea = TextInfoArea(self)
 		self.updateFont()
+		self.setWrapModeAndWidth()
 		self.document().blockCountChanged.connect(self.updateLineNumberAreaWidth)
 		self.cursorPositionChanged.connect(self.highlightCurrentLine)
 		self.document().contentsChange.connect(self.contentsChange)
 		if globalSettings.useFakeVim:
 			self.installFakeVimHandler()
 
+	def setWrapModeAndWidth(self):
+		if globalSettings.rightMarginWrap and (self.rect().topRight().x() > self.marginx):
+			self.setLineWrapMode(2)
```

Comment:
Please do not use magic numbers 1 or 2. Use `QTextEdit.WidgetWidth` and `QTextEdit.FixedPixelWidth` instead.